### DISCUSSION TITLE
Asp.Net WebApi & React js

### DISCUSSION THREAD

**Discussion by Shamoverse** [Sina-Soltani/Parbad#385] [Q&A]
با سلام و احترام 
بنده در Web Api استفاده کرده ام و به عنوان  Call Back Url  مستقیم متد Verify خود را صدا نزده ام و از روت React  استفاده شده است 
هنگام بازگشت از بانک به  React به ارور 405 برخورد میکنه و میگه اجازه دسترسی به وب سایت خارجی برای استفاده از فریم ورک داده نمیشود 
چگونه میتونم این موضوع رو کنترل کنم ؟ آیا در  React کانفیگ خاصی رو باید انجام بدم ؟ 
به دلیل ست کردن توکن  jwt  و بازگشت پاسخ به کاربر نیاز هست که از روت  Front استفاده شود لزوما؟
با تشکر

**Comment by Sina-Soltani** [comment]
درود
نمونه پروژه ها مشخص و واضح هستن.
نمونه برنامه Angular رو بررسی کنید و ببینید به چه صورت کار میکنه.

**Reply by Shamoverse** [reply]
با سپاس از زحمات شما
